ci/builders/README.md

```diff
@@ -154,231 +143,237 @@ An archive component is used to tell the recipes which artifacts are generated b
 
 Description of the fields:
 
-* **Name:** Used to identify the archive inside CAS.
-* **Base\_path:** The portion of the path to remove before uploading to its destination. In the example the base\_path **“out/host\_debug/zip\_archives”** will be extracted from the include path **"out/host\_debug/zip\_archives/linux-x64/artifacts.zip"** before uploading to GCS, e.g. &lt;bucket>/flutter/&lt;commit>/linux-x64/artifacts.zip.
-* **Type:** The type of storage to use. Currently only **“GCS”** and **“CAS”** are supported.
-* **Include\_paths:** A list of strings representing paths to artifacts generated by the build that need to be uploaded to a given destination.
-
-##### Drone\_dimensions
-
-A list of strings with key value pairs separated by an equal sign. These dimensions are used to select the bot where the sub-build will be running.
-
-```
-"drone_dimensions": [          "device_type=none",
+* **name:** - used to identify the archive inside CAS.
+* **base\_path:** - the portion of the path to remove from the full path before
+ uploading to its final destination. In the example the above the
+ base\_path **“out/host\_debug/zip\_archives”** will be removed from the
+ include path **"out/host\_debug/zip\_archives/linux-x64/artifacts.zip"**
+ before uploading to GCS, e.g.
+ &lt;bucket&gt;/flutter/&lt;commit>/linux-x64/artifacts.zip.
+* **Type:** - the type of storage to use. Currently only **“GCS”** and
+**“CAS”** are supported.
+* **Include\_paths:** - a list of strings with the paths to be uploaded to a
+given destination.
+
+#### Drone\_dimensions
```

Comment:
Is there a full description of the all the valid keys, and for each key what the valid values are somewhere? Can we link to that, or spell it out here?

Reply:
No, these are arbitrary values reported as dimensions by the swarming bot 
![image](https://github.com/flutter/engine/assets/54371434/7dd2506f-0171-4372-83ee-97111cddf291) added some additional info on how to find the values to use to select a bot.